src/wordlist.h

```diff
@@ -62,8 +91,18 @@ void wordlistDestroy(wordlist **);
  */
 char *wordlistChopHead(wordlist **);
 
-/// convert a wordlist to a SBufList
-SBufList ToSBufList(wordlist *);
```

Comment:
This removal is outside this PR scope, and I do not insist on it. I removed ToSBufList() because, if it stays, then its implementation should be modified to use the new iterator. Reverting the last branch commit (25e74e7) will re-introduce that modification.

This comment does not request any changes.

Reply:
I think wordlist has a functional place, and SBufList should take that place.
I agree the implementation of the converter should be moved to the iterator, so I'm okay with the removal,
reimplementation can happen when needed.